python/scannerpy/op.py

```diff
@@ -19,7 +19,15 @@ def __getattr__(self, name):
         elif name == 'Output':
             return lambda inputs: Op.output(self._db, inputs)
 
-        if not self._db._bindings.has_op(name):
+        has_op_args = self._db.protobufs.HasOpArgs()
+        has_op_args.op_name = name
+
+        try:
```

Comment:
This could also just be `self._db._try_rpc(lambda: self._db._master.HasOp(has_op_args)`